.gitignore

```diff
@@ -0,0 +1,3 @@
+/.sync
```

Comment:
This file should not be committed. These ignores are specific to your computer and should be ignored via a global ignore 

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
@navarr interesting approach, file .gitignore removed from PR.

Should we apply the same approach in https://github.com/wardenenv/warden/blob/main/.gitignore ?